app/configuration.py

```diff
@@ -54,6 +54,7 @@ class HoldingRegister:
     data_type: str
     scale: float
     address: list[int]
+    invert: bool = False
```

Comment:
I think, for the sake of avoiding ambiguity, I would call this something like `reverseSign`.
I know this is subjective, but it reads more clear to me what's the intention of the flag. WDYT?

Reply:
yes agree it could be more specific. How about `invert_sign` ?

Reply:
that works for me as well 👍